bmc/engine/check_settling_time.h

```diff
@@ -0,0 +1,170 @@
+/*
+ * check_settling_time.h
+ *
+ *  Created on: Nov 25, 2017
+ *      Author: Thiago Cavalcante
+ */
+
+#ifndef CHECK_SETTLING_TIME_H_
+#define CHECK_SETTLING_TIME_H_
+
+#include <Eigen/Dense>
+#include <Eigen/Core>
+#include <complex>
+#include <cstdlib>
+
+extern digital_system_state_space _controller;
+double new_state[LIMIT][LIMIT];
+
+using Eigen::MatrixXd;
+
+// Calculate the system's output
+double y_k(MatrixXd A, MatrixXd B, MatrixXd C, MatrixXd D, double u, int k, MatrixXd x0) {
+	MatrixXd y;
```

Comment:
You're not using the DS Verifier code style, which consists of 2 spaces for code indentation.